cmake/find/Gettext.cmake

```diff
@@ -182,17 +177,29 @@ set(XGETTEXT_GLADE_OPTIONS_DEFAULT
   "--language=Glade"
   "--omit-header"
 )
-set(XGETTEXT_INTLTOOL_OPTIONS
-  "--language=C"
-  "--keyword=N_:1"
+set(XGETTEXT_DESKTOP_OPTIONS
```

Comment:
On line 25 you could add your name to the people who modified the original file.